admin/src/scenes/public-support-center/form.js

```diff
@@ -67,20 +70,25 @@ export default function PublicSupportCenterForm({ setOpen, setSuccessMessage })
               touched={touched}
               rows="2"
             />
-            <SelectTag
-              name="step1"
-              options={Object.values(step1)}
-              title={"Ma demande"}
-              selectPlaceholder={"Choisir la catégorie"}
+            <Item
+              name="department"
+              type="select"
+              placeholder="Sélectionnez votre département"
+              values={values}
+              value={values.department}
               handleChange={handleChange}
-              value={values?.step1?.id}
+              title="Département"
+              options={departmentList.map((d) => ({ value: d, label: d }))}
```

Comment:
on peut les classer par ordre alphabetique, sinon les premiers se retrouvent tout en bas
```suggestion
              options={departmentList.map((d) => ({ value: d, label: d }))?.sort((a, b) => a.label.localeCompare(b.label))}
```